### PR TITLE
Adds octokit autopagination

### LINKED ISSUE #50: Check if we are using auto paginate for the branch fetch

### PR DESCRIPTION
**Pull request by mdesanti**
## Summary

Resolves #50

### CODE REVIEW

File: script/production_dump.sh [hunk 1]
```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
```
Comment (epintos) [inline script/production_dump.sh]: I uploaded another file with the old script, why don't you replace it?

File: config/initializers/octokit.rb [hunk 2]
```diff
@@ -4,3 +4,4 @@
   builder.adapter Faraday.default_adapter
 end
 Octokit.middleware = stack
+Octokit.auto_paginate = true
```
Comment (epintos) [inline config/initializers/octokit.rb]: Adding this only just works?

Comment (mdesanti) [inline config/initializers/octokit.rb]: https://github.com/octokit/octokit.rb#pagination

Comment (epintos) [inline config/initializers/octokit.rb]: nice!

**Comment by epintos** [comment]
👍

**Pull request merged**